lib/tmp.js

```diff
@@ -547,6 +547,24 @@ function _safely_install_listener() {
     }
   }
 
+  // windows does not support signals
+  // it'd never had won if it wasn't a major PITA
+  // with node v8.x and win 10 this is no longer an issue
+  if (process.platform == 'win32') {
+    var rl = require("readline").createInterface({
```

Comment:
Just a question, couldn't we simply pass over the `process`?

```javascript
const rl = require('readline').createInterface(process);
```

Reply:
trying this right now.

Reply:
this will fail, see existing builds on appveyor. we need to use the original code.